config/environments/production.rb

```diff
@@ -84,12 +84,12 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_controller.asset_host = "http://trase.earth"
-  config.relative_url_root = '/content'
+  config.action_controller.asset_host = "//staging.trase.earth"
```

Comment:
@santostiago please change this and redeploy. perhaps it's worth creating a "staging" settings for rails?